test/source/browser/controllable.ts

```diff
@@ -76,8 +76,9 @@ abstract class ControllableBase {
     this.log(`wait_all:7:${selectors.join(',')}`);
   };
 
-  public waitAny = async (selector: string | string[], { timeout = TIMEOUT_ELEMENT_APPEAR, visible = true }: { timeout?: number, visible?: boolean } = {}): Promise<ElementHandle> => {
-    return await this.waitAnyInternal(this.selsAsProcessedArr(selector), { timeout, visible });
+  public waitAny = async (selector: string | string[], properties: { timeout?: number, visible: true | undefined } | { timeout?: number } = {}): Promise<ElementHandle> => {
```

Comment:
Why `visible` type becomes `true | undefined`? Any problem with `visible?: boolean`?

Reply:
Never mind, Just figured out because visible `false` option should not be passed.

Reply:
In `waitAll` the meanings are:
`true` element must exist and be visible
`false` element must exist and be not visible.
`undefined` element must exist

`waitAny` doesn't support the option: element must exist and be not visible.
This PR makes `waitAny` in line with `waitAll` meanings